PR for SRVLOGIC-530: Add the section to upgrade Serverless Logic Operator from 1.35.0 to 1.36.0 (#93332)
openshift/openshift-docs: modules/serverless-logic-upgrade-1.36-finalizing-job-service.adoc

// Module included in the following assemblies:
// * serverless-logic/serverless-logic-upgrading-operator-from-1.35-to-1.36

:_mod-docs-content-type: PROCEDURE
[id="serverless-logic-upgrade-1.36-finalizing-job-service_{context}"]
= Finalizing the Job Service upgrade

After the {ServerlessOperatorName} is upgraded to version 1.36.0 you can optionally delete the old `ReplicaSet`.

.Procedure

> * (Optional) Identify the old `ReplicaSet` and delete it by running the following commands:

Reviewer: ```suggestion
* (Optional) Identify the old `ReplicaSet` resource and delete it by running the following commands:
```

Same comment about using only a single command per code block.

// Module included in the following assemblies:
// * serverless-logic/serverless-logic-upgrading-operator-from-1.35-to-1.36

:_mod-docs-content-type: PROCEDURE
[id="serverless-logic-upgrade-1.36-finalizing-job-service_{context}"]
= Finalizing the Job Service upgrade

> After the {ServerlessOperatorName} is upgraded to version 1.36.0 you can optionally delete the old ReplicaSet`.

Reviewer 2: ```suggestion
After the {ServerlessOperatorName} is upgraded to version 1.36.0 you can optionally delete the old `ReplicaSet`.
```

Reviewer 2: :point_up: nitpick

> .Procedure

Reviewer 2: @kaldesai 

For the jobs service we should apply the same pattern as for the Data Index
:point_down: 
![image](https://github.com/user-attachments/assets/6a5bdb69-0320-4b6b-ba3b-6993f0609ff8)

> After the {ServerlessOperatorName} is upgraded to version 1.36.0, a new ReplicaSet for the Job Service component is created automatically. To clean up older resources, you can delete the Job Service ReplicaSets created by the previous 1.35.0 version.

Reviewer 2: ```suggestion
After the {ServerlessOperatorName} is upgraded to version 1.36.0 you can optionally delete the old ReplicaSet`.
```

> . List all ReplicaSets for the Job Service by running the following command:

Reviewer 2: ```suggestion
. (Optional) Identify and the delete the old `ReplicaSet` running the following commands:
```

Author: @wmedvede 

For listing, the command is different. 
`oc get replicasets -o custom-columns=Name:metadata.name,Image:spec.template.spec.containers[*].image -n <target_namespace>`

For deleting the command is different: 
`oc delete replicaset <old_replicaset_name> -n <target_namespace>`

Then why are we removing these steps?

Reviewer 2: @kaldesai the case of the jobs service is analogous to the DataIndex ReplicasSet removal.
See attached screenshot.

People can optionally delete.

+
[source,terminal]
----
$ oc get replicasets -o custom-columns=Name:metadata.name,Image:spec.template.spec.containers[*].image -n <target_namespace>
----
+
.Example output

> [source,sql]

Reviewer 2: ```suggestion
[source,sql,subs="verbatim,quotes"]
```

----
Name                                                Image

> sonataflow-platform-jobs-service-1111111111         registry.redhat.io/openshift-serverless-1/logic-jobs-service-postgresql-rhel8:1.35.0

Reviewer 2: ```suggestion
*sonataflow-platform-jobs-service-1111111111         registry.redhat.io/openshift-serverless-1/logic-jobs-service-postgresql-rhel8:1.35.0*
```

Reviewer 2: I have added the *blabla* with hope that this particular line is displayed in bold in the produced outptut.
Help users to identify which is the like that must be used as reference for deletion.

sonataflow-platform-jobs-service-2222222222         registry.redhat.io/openshift-serverless-1/logic-jobs-service-postgresql-rhel8:1.36.0
----

> . Identify and delete the old ReplicaSet associated with version 1.35.0 by running the following command:

Reviewer 2: ```suggestion
```
